src/main/resources/assets/pswg/lang/pt_br.json

```diff
@@ -120,13 +122,14 @@
 	"item.pswg.qrikki_waffle": "Waffle Qrikki",
 	"item.pswg.salt_pile": "Pilha de Sal",
 	"item.pswg.titanium_ingot": "Lingote de Titanium",
+	"item.pswg.transparisteel_ingot": "Lingote de Transparisteel",
 	"itemGroup.pswg.main": "Galaxies: Parzi's Star Wars Mod",
 	"key.category.pswg": "Galaxies: Parzi's Star Wars Mod",
 	"key.pswg.lightsaber_toggle": "Alternar Sabre de Luz",
 	"key.pswg.ship_input_mode_override": "Substitui o modo de comando da nave",
 	"text.autoconfig.pswg.option.input": "Comando",
 	"text.autoconfig.pswg.option.input.@Tooltip": "Aqui você pode alterar os seus métodos de comando favoritos.",
 	"text.autoconfig.pswg.option.input.shipRollPriority": "Nave dará prioridade ao comando de rolar",
-	"text.autoconfig.pswg.option.input.shipRollPriority.@Tooltip": "Define se os comandos do mouse devem ser 'inclinar e guinar', ou 'inclinar e rolar'",
+	"text.autoconfig.pswg.option.input.shipRollPriority.@Tooltip": "Define se os comandos do mouse devem ser 'inclinar e guinar', ou 'inclinar e rolar',
```

Comment:
Ah, still need the trailing `"` at the end of the string

Reply:
Ah, you're right, damnit. Well, third times a charm, it's in the edit 2 commit